src/Features/Navigation/Menu.php

```diff
@@ -88,8 +88,9 @@ final public static function instance() {
 	public function init() {
 		add_action( 'admin_menu', array( $this, 'add_core_items' ) );
 		add_filter( 'admin_enqueue_scripts', array( $this, 'enqueue_data' ), 20 );
-		add_filter( 'add_menu_classes', array( $this, 'migrate_menu_items' ), 30 );
-		add_filter( 'add_menu_classes', array( $this, 'migrate_core_child_items' ) );
+
+		add_filter( 'admin_menu', array( $this, 'migrate_core_child_items' ) );
```

Comment:
This may fire too early.  Any plugins not using the default priority will be missed.  I think this should probably be `PHP_INT_MAX - 1` and the below `migrate_menu_items` should have a priority of `PHP_INT_MAX - 2`.

This doesn't guarantee that we get all plugins and I think that was part of the issue.  IIRC a plugin did manage to sneak through in testing which was our reason for moving to `admin_menu_classes`.  This may be more edge case though so I can support the move to `admin_menu` provided we tweak the priorities a bit.